apps/shipments/serializers.py

```diff
@@ -239,20 +239,27 @@ class PermissionLinkSerializer(serializers.ModelSerializer):
 
     class Meta:
         model = PermissionLink
-        exclude = ('shipment',)
-
-    def validate_expiration_date(self, expiration_date):
-        if expiration_date <= datetime.now(timezone.utc):
-            raise exceptions.ValidationError('The expiration date should be greater than actual date')
-        return expiration_date
+        fields = '__all__'
+        read_only = ('shipment',)
 
 
-class PermissionLinkCreateSerializer(PermissionLinkSerializer):
+class PermissionLinkCreateSerializer(serializers.ModelSerializer):
```

Comment:
I think `PermissionLinkCreateSerializer` can still inherit from `PermissionLinkSerializer` and you won't have to displace the validation method.

Reply:
I think I was originally having issues when creating it, but I think that was a seperate issue I was overlooking.

Reply:
Good catch @ClovisDj